logger.c:110:13: error: expected identifier or '(' before 'true'

Build failure on gcc15.
```
gcc -pipe -frecord-gcc-switches -Wall -g -O2 -flto=auto  -c logger.c
logger.c: In function 'main':
logger.c:110:13: error: expected identifier or '(' before 'true'
  110 |         int true = 1;
      |             ^~~~
logger.c:134:21: error: lvalue required as unary '&' operand
  134 |             (void *)&true, sizeof(true)))
      |                     ^
make: *** [Makefile:40: logger.o] Error 1
```

--- a/logger/logger.c
+++ b/logger/logger.c
@@ -107,7 +107,7 @@ static int drop_root(void)
 
 int main(int argc, const char * const *argv)
 {
-	int true = 1;
+	int one = 1;
 	int sock, new;
 	struct sockaddr_in addr;
 	socklen_t addrlen;
@@ -131,7 +131,7 @@ int main(int argc, const char * const *argv)
 		return log_error("socket");
 
 	if (setsockopt(sock, SOL_SOCKET, SO_REUSEADDR,
-	    (void *)&true, sizeof(true)))
+	    (void *)&one, sizeof(one)))
 		return log_error("setsockopt");
 
 	memset(&addr, 0, sizeof(addr));

